api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java

```diff
@@ -485,18 +485,6 @@ public Relationship saveRelationship(Relationship relationship) throws APIExcept
 	 */
 	@Override
 	public Relationship voidRelationship(Relationship relationship, String voidReason) throws APIException {
-		if (relationship.getVoided()) {
```

Comment:
@joachim-junior have you checked why travis is failing, did you run mvn clean install locally before pushing the code

Reply:
Yes, i did a mvn clean install before pushing.

Reply:
Am trying to see why travis is failing. Locally i did a mvn clean install and it built successfully.